packages/babel-plugin/src/index.js

```diff
@@ -41,11 +41,17 @@ const loadablePlugin = api => {
     }
 
     // `loadable.lib()`
-    return (
+    if (
       path.get('callee').isMemberExpression() &&
       path.get('callee.object').isIdentifier({ name: 'loadable' }) &&
       path.get('callee.property').isIdentifier({ name: 'lib' })
-    )
+    ) {
+      return true
+    }
+
+    // `lazy()`
+    return (path.get('callee').isIdentifier({ name: 'lazy' }))
```

Comment:
👍 
Sounds like a bug. It would be great to get this change as a separate PR

Reply:
Need to add some snapshot tests, how do I update snapshots?

Reply:
jest -I, or just delete old ones before running the test